src/vcDBF.h

```diff
@@ -0,0 +1,80 @@
+#ifndef vcDBF_h__
+#define vcDBF_h__
+
+#include "udResult.h"
+#include "udFile.h"
+
+enum vcDBF_DataMappedType
+{
+  vcDBFDMT_String,
+  vcDBFDMT_Float,
+  vcDBFDMT_Logical,
+  vcDBFDMT_Integer,
+  vcDBFDMT_Memo,
+  vcDBFDMT_Invalid
+};
+
+struct vcDBF;
+
+struct vcDBF_Record
+{
+  bool deleted;
+  union vcDBF_RecordField
+  {
+    char *pString;
+    double floaty;
+    bool logical;
+    int integer;
+  } *pFields;
+};
+
+template <typename T>
+void vcDBF_FlipEndian(T *pData)
```

Comment:
This doesn't need to be in the header, it's only used in the cpp.

Reply:
Yes, but this functionality is needed elsewhere, specifically for shapefiles